client/layout/guided-tours/config.js

```diff
@@ -11,13 +11,14 @@ import config from 'config';
  * Internal dependencies
  */
 import { getSelectedSite } from 'state/ui/selectors';
+import { isNewUser } from 'state/ui/guided-tours/selectors';
 
 const tours = {
 	main: {
 		meta: {
 			version: '20160601',
 			path: '/',
-			context: () => false,
+			context: isNewUser,
```

Comment:
Yep.

On Fri, Jul 22, 2016 at 12:20 PM Chris Blower <notifications@github.com>
wrote:

> In client/layout/guided-tours/config.js
> <https://github.com/Automattic/wp-calypso/pull/6926#discussion_r71863957>:
>
> >
> >  const tours = {
> >  	main: {
> >  		meta: {
> >  			version: '20160601',
> >  			path: '/',
> > -			context: () => false,
> > +			context: isNewUser,
>
> Yeah, I guess we should add a config( 'env' ) check here?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Automattic/wp-calypso/pull/6926/files/ad0d8949a8acabf49892a5675440fe1b521e2fac#r71863957>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAJMIkoynKszpRdoqaDNgoc6MYglYZYZks5qYKeDgaJpZM4JRA83>
> .
>